hook.php

```diff
@@ -282,13 +282,15 @@ function plugin_fields_rule_matched($params = [])
 
                         //retrieve computer
                         $agents_id = $params['input']['plugin_fusioninventory_agents_id'];
+                        /** @phpstan-ignore-next-line */
```

Comment:
`phpstan-ignore` is forbidden :)

In this case, you can check upstream whether the `PluginFusioninventoryAgent` class exists.

Reply:
L257, replace the `switch` with an `if` and add the condition: `class_exists(‘PluginFusioninventoryAgent’)` to remove `phpstan-ignore`